### DISCUSSION TITLE
How to translate default ValidationException message

### DISCUSSION THREAD

**Discussion by d0mhate** [pimcore/pimcore#8715] [Q&A / Support] [answered]
Hi.

How can i translate "Invalid data in field" the validation error message when saving an object?

Symfony version 4.4.20
Pimcore v6.8.10.

Thx.

**Comment by dvesh3** [comment, marked as answer]
We need to translate message in `Pimcore\Model\DataObject\ClassDefinition\Data::checkValidity()` 
https://github.com/pimcore/pimcore/blob/master/models/DataObject/ClassDefinition/Data/ManyToOneRelation.php#L404
we're happy to receive a PR 😉

**Reply by d0mhate** [reply]
I tried adding to administrative translations. Does not work. 
Do I need to override this method with class? Or is it something else?

**Reply by dvesh3** [reply]
I mean changing the message in the core and it should come from pimcore/CoreBundle/Resources/translations
